src/common/tusb_mcu.h

```diff
@@ -381,6 +385,8 @@
 
   #define CFG_TUSB_OS_INC_PATH_DEFAULT   freertos/
 
+  #define TUP_MCU_MULTIPLE_CORE   1
```

Comment:
@tore-espressif P4 is here, thanks for the tip. I will check to see if we could include soc_caps.h here

Reply:
ups, I missed that, thanks!